english/hospital_bed/hospital_bed.md

```diff
@@ -0,0 +1,8 @@
+Just then, a crazed ostrich smacks into you at full-speed.  Your phone flies out of your hands and you hit the pavement.  
+
+You wake up in the hospital tied down to your bed. IV's trickle into your veins. You shout, but you're all alone.  You see your phone at your side, but it's just out of reach.  What do you do?
+
+[Wait, and hope for the best] (..existential-dread/existential-dread.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.